src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java

```diff
@@ -245,12 +249,12 @@ public String getIdentificationString() {
 
     @Override
     public String getEntityTypeAsString() {
-        return "Feedback Question";
+        return ATTRIBUTE_NAME;
     }
 
     @Override
     public String getBackupIdentifier() {
-        return Const.SystemParams.COURSE_BACKUP_LOG_MSG + courseId;
```

Comment:
Don't forget to delete `Const.SystemParams.COURSE_BACKUP_LOG_MSG` if is not used.